doc/ref_model/chapters/chapter08.md

```diff
@@ -94,6 +94,59 @@ Figure 8-3 shows a conceptual Telco Operator Platform Architecture. The Cloud In
+
+<p align="center"><img src="../figures/rm-chap8-multi-cloud-interactions-02.png" alt="Multi-Cloud Interactions Model"></p>
+<p align="center"><b>Figure 8-4:</b> Multi-Cloud Interactions Model</p>
+
+The model defines the following core roles:
+- Communications Service Provider (CSP) - is the party responsible for providing end user service to their customer
+- Customer / User - are the parties that use the service (User) and establishes the business agreement for the service provision (Customer). For retail services the customer and user are the same party, while for enterprise services the Enterprise is the Customer (responsible for the business agreement) and its representatives are the Users.
+- Cloud Providers - are the parties providing the cloud services. These services could be any XaaS service. It could be that a CSP has an agreement with a SaaS Cloud, which in turn uses an IaaS Cloud Provider to deliver their service.
+
+The set of high level interactions cover:
+- Manage Account - covering Account, Users, Billing
+- Manage Connectivity - Public or Private Network, VPN Configuration, CSP Edge / Cloud Connection Configuration, Connection Security Profile
+- Manage Resource - Resource Pool Management, VM / VNF Management (CPU, Memory, Storage, Network), Image Repository Management, Storage Management, VNF / CNF LCM, Monitor Resources
+- Manage App/VNF - Image / Container / Registry Management, Deploy/Configure/Scale/Start/Stop App/VNF, Monitor App/VNFs
+- Transactions / Conversations - Use Communications Services, Use Edge Applications Services, Use Cloud Services
+
+<a name="8.2.4.2"></a>
+### 8.2.4.2 Stereo-Typical Scenarios
+A set of stereo-typical interactions cases are illustrated for the cases of a Simple Infrastructure-as-a-Service (IaaS) and Software-as-a-Service (SaaS), where deployment is on a Cloud Provider's centralised sites and/or Edge sites. The scenarios help highlight needs for the Cloud Service Broker and Cloud Resources Broker (as per Figure 8.3) and hence extent of orchestration required to manage the interactions.
+
+<p align="center"><img src="../figures/rm-chap8-multi-cloud-interactions-simple-stereo-types-02.png" alt="Simple Stereo-Type Interactions"></p>
+<p align="center"><b>Figure 8-5:</b> Simple Stereo-Type Interactions</p>
+
+The following patterns are visible:
+- For IaaS Cloud Integration:
+  - Cloud behaves like VIM and so needs Orchestration
+  - Depending on whether the cloud is accessed via public internet or private connection will change the extend of the Connectivity Management
+- For SaaS Cloud Integration:
+  - Cloud behaviors is like subscription management, so avoids need for complex orchestration
```

Comment:
```suggestion
  - Cloud behaves like a running application/service and requires subscription management, and complex orchestration of the app/service and underlying resources
```

Reply:
Thank you @pgoyal01.
I have added to this sentence:

Cloud behaves like a running application/service and requires subscription management, and complex orchestration of the app/service and underlying resources is managed by SaaS provider with the User relieved of having to provide direct control of resources

As what I was saying is that at its most basis the consumption of SaaS takes away the users need to have to worry about the resource management (it is all handled behind the interface).

I know that this is a generalisation, as is most of the "stereo-typical" analysis but it is important on how the APIs would be presented to User, as level of control with SaaS is highly constrained.

Reply:
@jbhartley, "level of control with SaaS is highly constrained": I couldn't agree more.  Especially from the operations and support perspective, the aspect of which often people forget, thinking of SaaS as a piece of software like MS Word.  I have some recent first hand experience ...

Reply:
@walterkozlowski acknowledged. Thank you Walter